static/docs/src.html

```diff
@@ -139,6 +140,14 @@
         }
       }
 
+      function fixTableAlign() {
+        for(const elem of document.querySelectorAll("table")) {
+            if(elem.innerHTML.includes('align="left"')){
+                elem.innerHTML = elem.innerHTML.replace(/align="left"/g, "");
+            }
+        }
```

Comment:
It will be better if we replace the align attribute with a class. Changing innerHTML is very performance heavy (entire string needs to parsed again as HTML and converted to DOM tree), and also, it doesn't show the intention explicitly. So, we can do:
```js
if (elem.hasAttribute("align")) {
  const className = `align-${elem.getAttribute("align")}`;
  elem.classList.add(className);
  elem.removeAttribute("align");
}
```

Also, we can select only those elements that have align attribute, instead of all tables. Then we can remove the `if` statement also.
```js
document.querySelectorAll("[align]")
```

Reply:
Or, we can add inline `style` attributes. Then we don't need to classes either.

Reply:
I'm thinking we should use inline style instead. There are like 2-3 instances total.